config/initializers/active_job_uniqueness.rb

```diff
@@ -23,7 +23,7 @@
 
       # Digest method for lock keys generating. Expected to have `hexdigest` class method.
       #
-      # config.digest_method = OpenSSL::Digest::MD5
+      config.digest_method = Digest::SHA256
```

Comment:
We need to test active-job-uniqueness working especially make sure Redis max length not reached.